kbase-extension/scss/partials/_job-status.scss

```diff
@@ -134,91 +133,52 @@
     &--object,
     &--status {
       @include table-cell;
-    }
-  }
 
-  &__icon {
-    margin: 4px;
-
-    /*
-      .kb-job-status__icon--completed
-    */
-    &--completed {
-      color: use-color('success-dark');
-    }
-
-    /*
-      .kb-job-status__icon--error,
-      .kb-job-status__icon--does_not_exist
-    */
-    &--error,
-    &--does_not_exist {
-      color: use-color('error');
+      white-space: nowrap;
     }
+  }
 
-    /*
-      .kb-job-status__icon--terminated
-    */
-    &--terminated {
-      color: use-color('warning-dark');
-    }
+  /* .kb-job-status__param_list */
+  &__param_list {
+    @include body-text;
+    list-style-type: none;
+    margin: 1rem 0;
+    padding: 0;
+  }
 
-    /*
-      .kb-job-status__icon--queued,
-      .kb-job-status__icon--created,
-      .kb-job-status__icon--estimating,
-      .kb-job-status__icon--running
-    */
-    &--queued,
-    &--created,
-    &--estimating,
-    &--running {
-      color: use-color('info');
-    }
+  /* .kb-job-status__log_container */
+  &__log_container {
+    background-color: use-color('info-lighter');
+    border-bottom: 1px solid use-color('base-lightest');
+    padding: 1rem;
   }
 
   /* .kb-job-status__summary */
   &__summary {
     color: use-color('ink');
     font-weight: 700;
+  }
 
-    /* .kb-job-status__summary--completed */
-    &--completed {
-      color: use-color('success-dark');
-      font-weight: 700;
+  @each $key, $value in $job_status_colours {
```

Comment:
programmatically generate a set of classes for each job state with the appropriate text colour.